src/js/ui/shared/round/view/roundView.jsx

```diff
@@ -0,0 +1,382 @@
+"use strict";
```

Comment:
Sorry about that. Ever since these files started showing up I've been adding specific files or doing a `git diff` before a `git add -A` to make sure there are none. Not sure how this one slipped through. Will be more vigilant going forward.